samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -10,43 +10,52 @@ class TeamsMessagingExtensionsActionBot extends TeamsActivityHandler {
                 return createCardCommand(context, action);
             case 'shareMessage':
                 return shareMessageCommand(context, action);
+            case 'webView':
```

Comment:
I can't comment in the right spot since it isn't touched in this PR, but on line 7, `handleTeamsMessagingExtensionSubmitAction()` needs to be `async`